src/operations.jl

```diff
@@ -4,25 +4,75 @@ hash(c::TransparentGray, hx::UInt) = hash(alpha(c), hash(gray(c), hx))
 hash(c::AbstractRGB, hx::UInt) = hash(blue(c), hash(green(c), hash(red(c), hx)))
 hash(c::TransparentRGB, hx::UInt) = hash(alpha(c), hash(blue(c), hash(green(c), hash(red(c), hx))))
 
+# range
+range{T<:HSV}(::Type{T}) = (360,1,1)
```

Comment:
This seems like a rather different meaning of `range`. I think I'd prefer a separate name...`gamutmax`? Sort of like `typemax`?